### PR TITLE
Added a new component for login page

### PR DESCRIPTION
**Pull request by PilgrimViis**

### CODE REVIEW

**Comment by PilgrimViis** [comment]
I took bye's registration page and have updated it. I've tried to merge his branch into mine, but I've failed =(  I've made a redesign if you don't mind. I want to make "Login" and "Register" buttons disabled, until the whole form would be valid. It doesn't look good with just border shadowing. Also, I've tried to take byes slider, but it looked awful on mobile, so I've switched to bootstrap switch.
image

File: Client/src/app/_models/loginInfo.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface LoginInfo {
+  email?: string;
```
Comment (chrisK00) [inline Client/src/app/_models/loginInfo.ts]: I think you are currently able to login either using email or username so having this named username might be more appropiate

File: Client/src/app/_models/registerInfo.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export interface RegisterInfo {
+  username?: string;
+  dateOfBirth?: string;
```
Comment (chrisK00) [inline Client/src/app/_models/registerInfo.ts]: Date type

**Review by chrisK00 — CHANGES_REQUESTED** [review]
Very readable code :tada:

**Review by chrisK00 — APPROVED** [review]

**Pull request merged**